tools/sof-combinatoric.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
```

Comment:
we have PCM with playback and capture capabilities, what if we want a combination of 0, 0 ?

Reply:
Here, I suggest use the index & output is index combine, not the pipeline id  
like help to say  
```
for example:
    target: C 4 2
    parameter: -t c -n 4 -p 2
    output: "0,1 0,2 0,3 1,2 1,3 2,3"
    target: P 4 2
    parameter: -t p -n 4 -p 2
    output: "0,1 0,2 0,3 1,0 1,2 1,3 2,0 2,1 2,3 3,0 3,1 3,2"
```
so when you dump the id 0 for both, you will got index of them is '0, 1'

Reply:
Please add some inline comment to tell others, people like me may make a mistake, and regard the number as pipeline id